### PR TITLE
Ported MAP preferences from Vore + custom preview backgrounds.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
immagine

Todo:

- [x] Figure out the big empty space thing

- [x] SQL

- [ ] Maybe move the map

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by NonQueueingMatt** [comment]
!wip

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -152,9 +157,8 @@ datum/preferences
 	var/savefile/loaded_character
 	var/datum/category_collection/player_setup_collection/player_setup
 
-	var/dress_mob = TRUE
-
-
+	var/icon/bgstate = "000"
```
Comment (mikomyazaki) [inline code/modules/client/preferences.dm:160]: Isn't this default state an icon_state not an icon?

Comment (NonQueueingMatt) [reply code/modules/client/preferences.dm:160]: lol you're right

**Pull request merged**